docs/source/ipex/inference.mdx

```diff
@@ -14,7 +14,7 @@ Optimum Intel can be used to load models from the [Hub](https://huggingface.co/m
 
 ## Loading
 
-You can load your model and apply IPEX optimizations (apply torch.compile for non-generation tasks). For supported architectures like LLaMA, BERT and ViT, further optimizations will be applied by patching the model to use custom operators.
+You can load your model and apply IPEX optimizations (apply torch.compile except text-generation tasks). For supported architectures like LLaMA, BERT and ViT, further optimizations will be applied by patching the model to use custom operators.
```

Comment:
I don't understand what you mean here, I see torch.compile being applied to `text-generation` task, why does it say "except text-generation tasks" here ?

Reply:
Actually, we didn't apply torch.compile in text-generation task which means IPEXModelForCausal LM doesn't have torch.compile in __init__. And generation tasks are also excluded in IPEXModel.__init__ when calling torch.compile